addOns/authhelper/src/main/java/org/zaproxy/addon/authhelper/AuthTestDialog.java

```diff
@@ -236,7 +235,7 @@ private JPanel getResultsPanel() {
                             Constant.messages.getString(
                                     "authhelper.auth.test.dialog.results.verif")),
                     LayoutHelper.getGBC(0, y, 1, 1L, insets));
-            resultsPanel.add(verifIdLabel, LayoutHelper.getGBC(1, y++, 1, 1L, insets));
+            resultsPanel.add(verifIdLabel, LayoutHelper.getGBC(1, y, 1, 1L, insets));
```

Comment:
This should be reverted, it avoids the need to touch this code when more fields are added.